third_party/openocd/BUILD

```diff
@@ -28,6 +28,12 @@ genrule(
     visibility = ["//visibility:public"],
 )
 
+filegroup(
+    name = "jtag_ftdi_cfg",
+    srcs = ["//util/openocd/board/cw340_ftdi.cfg"],
```

Comment:
@pamaury how this should be done?
when I run the command `./bazelisk.sh outquery //third_party/openocd:jtag_ftdi_cfg` I get the error:
```
no such package 'util/openocd/board/cw340_ftdi.cfg': BUILD file not found in any of the following directories. Add a BUILD file to a directory to mark it as a package.

```

Reply:
I think the best option is not to create a entry `jtag_ftdi_cfg` in `third_party/openocd`. Especially this the file you created is not part of `openocd` itself. You can just create the file as you did and access it through `//util/openocd/board/cw340_ftdi.cfg` . If you really want to do that, first I think you would need to write `//util/openocd/board:cw340_ftdi.cfg` (colon not slash) and create a `BUILD` file in `util/openocd/board` with an export statement similar to that of `util/openocd/target`